client/ayon_hiero/api/pipeline.py

```diff
@@ -97,7 +151,7 @@ def containerise(track_item,
     """
 
     data_imprint = OrderedDict({
-        "schema": "openpype:container-2.0",
+        "schema": "ayon:container-2.0",
```

Comment:
There is actually a 3.0 schema for AYON, see: https://github.com/ynput/ayon-core/blob/ed39b769cfc47bf3e12cfe86987c37a018531d8d/client/ayon_core/pipeline/schema/container-3.0.json#L4

I wonder if this should update along.
```suggestion
        "schema": "ayon:container-3.0",
```